update.sh

```diff
@@ -29,7 +29,7 @@ case "$ruby" in
 	rubinius)
 		ruby="rbx"
 		exts=(tar.bz2)
-		downloads_url="http://releases.rubini.us/"
+		downloads_url="https://releases.rubini.us/"
```

Comment:
Does this work though?

Reply:
Turns out Rubinius support was just completely broken. I've fixed it now.